docs/security/verify-installers.md

```
@@ -28,24 +28,52 @@ If shown, you can ignore the following warning:
-   - Version 1.18.0: `3300052c3561155e2baf361702000000052c35`
-   - Versions 1.6.11 to 1.17.1: `00d77e4f8b938f56ae265cd08e9193490c`
-   - Versions 1.4.12 to 1.6.10: `63c45bff1a148d60ed2994d3a2639034`
-   - Versions up to 1.4.11: `1a360f3933964c71f14e8754d94615d4`
+4. Click on View Certificates and select the field `Thumbprint`.
+
+<Image src="/img/security/verify-win-installer.png" srcset=" /img/security/verify-win-installer 1x, /img/security/verify-win-installer@2x.png 2x" alt="How to check the code signing certificate on Windows" width="1316" height="767" />
+
+### Certificate thumbprints for all Cryptomator versions {#windows-all-versions}
+
+Every Cryptomator installer is signed with a certificate. A certificate is identified by its thumbprint. The signing certificate changed over time and the following table shows for each version the certificate thumbprint:
+| Version(s)          | Certificate Thumbprint                     |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)